packages/gravity-ui-web/src/sass/05-components/03-organisms/_card-basic.scss

```diff
@@ -16,12 +17,55 @@
   }
 
   img {
+    display: block;
     width: 100%;
-    height: 16rem;
+    height: $card-image-height;
     object-fit: cover;
   }
 
-  > a img {
-    @include grav-decoration-notch-in;
+  > a {
+    display: block;
+    position: relative;
+
+    > img {
+      @include grav-decoration-notch-in;
+    }
+
+    > svg {
+      @include grav-color-grp-a-apply('fill', 'neutral', true);
+
+      width: 4rem;
+      height: $card-image-height;
+      position: absolute;
+      top: 0;
+      left: 50%;
+      margin: 0;
+      opacity: 0;
+      transform: translate(-50%, 0);
+      transition: opacity 0.1s ease-in-out;
```

Comment:
Would it make sense to have local SASS vars for the transition duration and timing function?

If they were ever changed, there's less risk of the overlay and arrow accidentally drifting out of sync.

Feel free to ignore though. I'm probably overthinking / overengineering as usual :-P

Reply:
We could define transitions and timings at a different level in gravity. That might be nice, and then just use a mixin to apply them.

(Not necessarily to keep the arrow and background transitions in sync, as I can see a future need for them to be out of sync on purpose.)

@james-nash maybe i could create a `tools/_transitions.scss` and put mixins in there (and for now, the variables they use). Does that sound right to you?